pyproject.toml

```diff
@@ -86,40 +86,40 @@ nameparser = "^1.1.3"
 eyecite = "*"
 scipy = "^1.14.1"
 scikit-learn = "^1.5.2"
-numpy = "^2.1.1"
+numpy = "^2.1.2"
 datasketch = "^1.6.5"
-PyStemmer = "^2.2.0.1"
+PyStemmer = "^2.2.0.3"
 factory-boy = "^3.3.1"
 django-elasticsearch-dsl = "^8.0"
 django-override-storage = "^0.3.2"
 django-environ = "^0.11.2"
 judge-pics = "^2.0.5"
-django-admin-cursor-paginator = "^0.1.5"
-sentry-sdk = {extras = ["celery", "django"], version = "^2.15.0"}
+django-admin-cursor-paginator = "^0.1.6"
+sentry-sdk = {extras = ["celery", "django"], version = "^2.16.0"}
 selenium = "^4.25.0"
 ipython = "^8.28.0"
-time-machine = "^2.15.0"
+time-machine = "^2.16.0"
 dateparser = "1.2.0"
 types-dateparser = "^1.2.0.20240420"
-uvicorn = {extras = ["standard"], version = "^0.31.0"}
+uvicorn = {extras = ["standard"], version = "^0.31.1"}
 daphne = "^4.1.2"
 httpx = {extras = ["http2"], version = "^0.27.2"}
-django-model-utils = "^4.5.1"
-django-permissions-policy = "^4.21.0"
+django-model-utils = "^5.0.0"
```

Comment:
Changelog: https://github.com/jazzband/django-model-utils/blob/5.0.0/CHANGES.rst#500-2024-09-01

Reply:
@albertisfu, this looks like a big version bump. Can you confirm it's OK?

Reply:
This it's good. But found an issue related to `django-storages` described in detail below.